.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: Continuous Integration (CI)
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout to the repository
+        uses: actions/checkout@v2
+
+      - name: Set up NodeJS environment
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
```

Comment:
What do you think of using the hard-coded 16 version? We could have a .nvmrc file instead:
```
with:
          # Reads the node version from the .nvmrc file itself 
          node-version-file: '.nvmrc'
          # The node version can be hard-coded 
          # node-version: 16
```

Reply:
I'm ok with having the node version hard coded

Reply:
Setting node version to 18